packages/mask/src/plugins/Wallet/SNSAdaptor/GasSettingDialog/Prior1559GasSetting.tsx

```diff
@@ -167,7 +167,7 @@ export const Prior1559GasSetting: FC<GasSettingProps> = memo(
         const handleConfirm = useCallback(
             (data: zod.infer<typeof schema>) => {
                 onConfirm({
-                    gasLimit: data.gasLimit,
```

Comment:
I don't know why this isn't a type error before. But data.gasLimit is a string, but gasLimit in onConfirm is number. I won't touch it in this PR. cc @guanbinrui